__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+const { forEach } = require("../db/data/test-data/articles");
+
+beforeEach(() => {
+	return seed(data);
+});
+afterAll(() => {
+	return db.end();
+});
+
+describe("GET/api/topics", () => {
+	it("Response has to be 200", () => {
+		return request(app).get("/api/topics").expect(200);
+	});
+	it("returns the correct properties on the correct number of topics", () => {
```

Comment:
This test is looking at 2 different behaviours - it could be beneficial to separate these into 2 tests